stjude/proteinpaint
# Wsi thumbnails

## Description

Removed ol map switcher and implemented custom thumbnail switcher. 

To test:
Load: http://localhost:3000/?noheader=1&mass={%22genome%22:%22mm10%22,%22dslabel%22:%22nintendoPublic%22}

Click on the sample view, the new image switcher should appear:

<img width="753" alt="Screenshot 2024-08-16 at 11 48 54 AM" src="https://github.com/user-attachments/assets/01d248ba-41ef-4665-ae34-3e93413b1d71">


## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
